app/overlays/mtl/fpga_overlay.conf

```diff
@@ -1,2 +1,3 @@
 CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=19200000
 CONFIG_DAI_DMIC_HW_IOCLK=19200000
+CONFIG_XTENSA_CCOUNT_HZ=24000000
```

Comment:
is this correct in all cases? I clearly remember an FPGA image running faster for DMIC tests.

Reply:
My mistake, I copied the wrong value, it should be 40 MHz.